lib/rules/lint-attribute-indentation.js

```diff
@@ -408,6 +424,8 @@ module.exports = class AttributeSpacing extends Rule {
       let result = {
         maxLength: 80,
         indentation: 2,
+        blocklessMustacheEnd: 'new-line',
```

Comment:
Hmm, why is this only for blockless? Seems like the same concept applies here:

```hbs
{{#some-thing
  whatever=qux
  stuff=goes-here
  onclick=(action 'handleClick')}}
  stuff here
{{/some-thing}}
```

vs

```hbs
{{#some-thing
  whatever=qux
  stuff=goes-here
  onclick=(action 'handleClick')
}}
  stuff here
{{/some-thing}}
```

Reply:
Also, how about `mustacheOpenEnd`?

Reply:
Thanks @rwjblue, my understanding by reading https://github.com/ember-template-lint/ember-template-lint/issues/267 is that this issue is about validating the closing brackets of mustache expressions only. Currently there is no validation for the closing brackets of block elements (is this a bug?).
I'm more than happy to handle the validation of block elements as well. Should we handle this with a separate configuration parameter to support the current behavior?
These parameters would be (trying to summarize all feedback):
* `mustacheEndOpen`: `true|false` (`true` `}}` on a new line, `false` next to the last attribute);
* `blockExpressionEndOpen`: `true|false` (`true` `}}` on a new line, `false` next to the last attribute);
* `elementEndOpen`: `true|false` (`true` `}}` on a new line, `false` next to the last attribute).

How does that sound?

Reply:
> Currently there is no validation for the closing brackets of block elements (is this a bug?).

Ya, I am aware. The main issue is that its fairly tricky. For mustache statements we know exactly where the `}}` is (because the `node.loc.end` represents it), but for blocks we have to find it (likely using the source) since `node.loc.end` is actually referencing the block closing tag.

I do really want to have coverage for the "block end open tag", happy to treat that as another issue after landing this (to keep moving forward with the incremental improvements and whatnot) if you think that works better...

> Should we handle this with a separate configuration parameter to support the current behavior?

I _do_ feel strongly that we should validate block open, but I _do not_ feel strongly around the config. I think my gut feeling would be that an app would want both blockless and blocks to be similarly formatted (and therefore we would only need a single config for those), but again I don't really feel very strongly there...